### ISSUE TITLE
ShareSafe : A web app to track fake news 

### ISSUE THREAD

**Issue by annu12340** [Zubi-io/Project-Ideas#4]
**Before you start, please follow this format for your issue title**:  
ShareSafe

## ℹ️ **ShareSafe**:  
_Please complete all applicable._

- **Project Name**: ShareSafe
- **Short Project Description**:  A blockchain platform that allows users to send and receive messages securely
- **Any supporting Link**: [Github repo](https://github.com/annu12340/15-days-of-code/tree/master/annujolly17%40gmail.com)
- **Labels**: Blockchain, web-app, beginner

## 🔥 Your Pitch
We live in a world, where all the information is available at our fingertips.  But as the information at our hands grow, it begins difficult to distinguish between the right and fake information
Today fake news is spreading as fast as wildfire. As people continue to share and re share these fake news, it becomes difficult to keep track of it and trace its origin. But using a blockchain network would help us combat this situation. 

I would like to create a miniature version of the medium app, where users can create post, like, dislike, and share them. The platform also allows users to follow each other. When a post is created, a new node is added in the blockchain and when the post is shared by the user, it would be recorded as a transaction in the bc network. 

For instance, in the recent days the fake news 'standing in the sun kill coronavirus'  has been rapidly spreading in FB and Whatsapp, even though there is no scientific evidence to back up this claim. Due to a large number of resharing, it is difficult to track the path of this fake news. But using blockchain, we could easily implement a fake news detector

In short, I would be to build a platform where users can share information safely and securely

## 🔦 Any other specific thing you want to highlight?
_(Optional)_

## ✅ Checklist

**Before you post the issue**:
- [x] You have followed the issue title format.
- [x] You have mentioned the correct labels.
- [x] You have provided all the information correctly.
- [x] Priority
  - [ ] **#1** : High
  - [x] **#2** : Medium
  - [ ] **#3** : Low
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I have [joined the telegram group](https://t.me/zubi_io).
